gen2-record-replay/libraries/depthai_record.py

```diff
@@ -242,3 +270,193 @@ def stream_out(name, fps, out, noEnc=False):
         self.pipeline = pipeline
         return pipeline, nodes
 
+
+class Mcap:
```

Comment:
Move the class to libraries/depthai_mcap.py, and rename it to `DepthAiMcap` (similar to the `DepthAiBags`)